openmrs-client/src/main/java/org/openmrs/mobile/activities/formdisplay/FormDisplayMainPresenter.java

```diff
@@ -379,7 +390,7 @@ public void createEncounter(boolean isEligible, boolean isValid, String mMessage
                 identifier.setIdentifier(identifier.getIdentifier()+mPatientID);
                 List<PatientIdentifier> identifiers = new ArrayList<PatientIdentifier>();
                 identifiers.add(identifier);
-                if (mFormname.equals("Risk Stratification Adult") || mFormname.equals("Risk Assessment Pediatric")) {
```

Comment:
@emadubuko please update these to "equalsIgnoreCase" to avoid issues where the forms are named in the same case